packages/amplify-graphql-relational-transformer/package.json

```diff
@@ -27,6 +27,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "@aws-amplify/graphql-auth-transformer": "^0.12.0",
```

Comment:
Looks like this dependency is not necessary since it only imports the feature flag mock which already exists in relational transformer package